src/user-guide/running-workflows/workflow-run-modes.rst

```diff
@@ -0,0 +1,113 @@
+.. _workflow-run-modes:
+
+Workflow Run Modes
+------------------
+
+Cylc can run a workflow without running the real jobs, which may be
+useful when developing workflows. Tasks submitted
+using :ref:`workflow-run-modes.dummy` and
+:ref:`workflow-run-modes.simulation` can be used to understand how different outcomes
+and run lengths will affect the workflow.
+
+To start a workflow in one of these modes:
+
+.. code-block:: console
+
+   $ cylc play --mode=dummy <workflow-id>
+   $ cylc play --mode=simulation <workflow-id>
+
+.. admonition:: Limitations
```

Comment:
Are these really limitations as such, or just points to be aware of. IMO a limitation is something that you might legitimately expect should work but doesn't.

Reply:
I don't really care what we call it - and to be fair, I think that the first point is something which does fit your definition of a limitation - superficially one might expect to be able to re-run in live mode.

```suggestion
.. note::
```

Reply:
Fair enough, happy to leave it as a "limitation".